src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -1113,18 +1115,14 @@ private void AddReadOnlyIfNeeded()
 
         private void AddParameterRefKindIfNeeded(IParameterSymbol symbol)
         {
-            if (format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeParamsRefOut))
+            if (format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeModifiers))
             {
                 if (symbol.ScopedKind == ScopedKind.ScopedRef &&
-                    !symbol.IsThis &&
-                    format.CompilerInternalOptions.IncludesOption(SymbolDisplayCompilerInternalOptions.IncludeScoped))
+                    symbol.RefKind != RefKind.Out &&
```

Comment:
`out` parameters are implicitly `scoped` (at least from modules compiled with `-langversion:11` or `net7.0`) so `scoped` is dropped from `out` parameters here, regardless of whether `scoped` appeared explicitly in source.